config/deploy.rb

```diff
@@ -11,7 +11,8 @@
 
 append :linked_files,
   'config/application.yml',
-  'config/database.yml'
+  'config/database.yml',
+  'config/credentials/production.yml.enc'
```

Comment:
```suggestion
  'config/credentials/production.key'
```

Double checking cause of this discrepancy -

> The production key has already been added to the server at /srv/jobapps/shared/config/credentials/production.key

This is definitely 100% correct?

Reply:
just looked it up and indeed I misunderstood